dockerfiles/is/README.md

```diff
@@ -13,25 +13,26 @@ git clone https://github.com/wso2/docker-is.git
 >The local copy of the `dockerfiles/is` directory will be referred to as `IS_DOCKERFILE_HOME` from this point onwards.
 
 ##### 2. Add JDK, WSO2 Identity Server distribution and MySQL connector to `<IS_DOCKERFILE_HOME>/files`
-- Download [JDK 1.8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- Download [JDK 1.8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) `.tar.gz` for Linux x64
 and extract it to `<IS_DOCKERFILE_HOME>/files`.
 - Download the WSO2 Identity Server 5.5.0 distribution (https://wso2.com/identity-and-access-management)
-and extract it to `<IS_DOCKERFILE_HOME>/files`. <br>
-- Once both JDK and WSO2 Identity Server distributions are extracted it may look as follows:
+and extract it to `<IS_DOCKERFILE_HOME>/files`.
+- Download [MySQL Connector/J](https://dev.mysql.com/downloads/connector/j/) v5.1.45, extract it, and copy the `*-bin.jar` to `<IS_DOCKERFILE_HOME>/files`.
```

Comment:
@afeld you have referred here to MySQL Connector version 5.1.45 here. But in line 25 you have referred to `mysql-connector-java-5.1.46-bin.jar`.

It would be great if you could fix this inconsistency.